test/integration/model/findOne.test.js

```diff
@@ -292,6 +292,22 @@ describe(Support.getTestDialectTeaser('Model'), () => {
           expect(user.username).to.equal('longUserNAME');
         });
       }
+
+      if (dialect === 'postgres') {
+        it('should allow case-sensitive find on TSVECTOR type', async function () {
+          const User = this.sequelize.define('UserWithCaseInsensitiveName', {
+            username: Sequelize.TSVECTOR
+          });
+
+          await User.sync({ force: true });
+          await User.create({ username: 'longUserNAME' });
+          const user = await User.findOne({
+            where: { username: 'longUserNAME' }
+          });
+          expect(user).to.exist;
+          expect(user.username).to.equal("'longUserNAME'");
```

Comment:
No idea why the result came out wrapped with quotes, no idea how to trim those via the TSVECTOR class

Reply:
That's totally normal the text values are retrieved as words hence these words are surrounded by quotes,
sees also here: https://www.postgresql.org/docs/10/datatype-textsearch.html